contrib/images/desmos-env/Dockerfile

```diff
@@ -8,8 +8,8 @@ ARG UID=1000
 ARG GID=1000
 
 USER ${UID}:${GID}
-VOLUME [ "/desmos" ]
-WORKDIR /desmos
+VOLUME [ "/.desmos" ]
+WORKDIR /.desmos
```

Comment:
App wiring tool tries to create a default dir `/.desmos` at first step, docker-compose is not root permission so it can not create the dir. It can be fixed by changing `/desmos` into `/.desmos` inside dockerfile and testnet generated dir.